### PR TITLE
Download up-to-date standard as JSON

### LINKED ISSUE #5: Update to use newest NMW codes
Two options for this:
1. (Easy) copy over the JSON from the main repo
2. (Slightly harder) add code to download the JSON from the main repo
    - This is preferred as it means the app is always up-to-date

### PR DESCRIPTION
**Pull request by miklasr**
**Changes**
- Add code to download newest NMW codes from main repo
- Remove old NMW code table

Closes #5

### CODE REVIEW

**Comment by Ashwin-MJ** [comment]
I got the following error in the Metro server upon start up:

```
Error: Problem validating fields in app.json. See https://docs.expo.io/workflow/configuration/
 • Field: ios.bundleIdentifier - 'ios.bundleIdentifier' should be a iOS bundle identifier notation unique name for your app. For example, `host.exp.expo`, where `exp.host` is our domain and `expo` is our app name..
```

**Comment by Ashwin-MJ** [comment]
I got the following error in the Metro server upon start up:

```
Error: Problem validating fields in app.json. See https://docs.expo.io/workflow/configuration/
 • Field: ios.bundleIdentifier - 'ios.bundleIdentifier' should be a iOS bundle identifier notation unique name for your app. For example, `host.exp.expo`, where `exp.host` is our domain and `expo` is our app name..
```

File: storage.tsx [hunk 1]
```diff
@@ -25,8 +54,9 @@ class Storage {
   }
 
   // Create and populate database tables
-  createTable() {
-    {
+  async createTable(downloadCompletedCallback: Function) {
+    downloadNMWCodes(NMWSTANDARD).then((nmwTable) => {
+      console.log(nmwTable);
```
Comment (Ashwin-MJ) [inline storage.tsx]: Could we remove this log before publishing?

**Review by Ashwin-MJ — COMMENTED** [review]
Works perfectly on both Android and iOS! One log to remove, and I think we should figure out how to resolve the previous error I mentioned

**Comment by miklasr** [comment]
> I got the following error in the Metro server upon start up:
> 
> ```
> Error: Problem validating fields in app.json. See https://docs.expo.io/workflow/configuration/
>  • Field: ios.bundleIdentifier - 'ios.bundleIdentifier' should be a iOS bundle identifier notation unique name for your app. For example, `host.exp.expo`, where `exp.host` is our domain and `expo` is our app name..
> ```

I don't get an error for that. Is there any chance you could look into that?

File: storage.tsx (hunk 1 above)
Comment (miklasr) [reply storage.tsx]: Good catch, I've removed it.

**Comment by Ashwin-MJ** [comment]
> > I got the following error in the Metro server upon start up:
> > ```
> > Error: Problem validating fields in app.json. See https://docs.expo.io/workflow/configuration/
> >  • Field: ios.bundleIdentifier - 'ios.bundleIdentifier' should be a iOS bundle identifier notation unique name for your app. For example, `host.exp.expo`, where `exp.host` is our domain and `expo` is our app name..
> > ```
> 
> I don't get an error for that. Is there any chance you could look into that?

I got it when running on both Android and iOS

**Comment by Ashwin-MJ** [comment]
Fixed it, looks like `bundleIdentifier` didn't like the use of underscores

**Review by Ashwin-MJ — APPROVED** [review]

**Pull request merged**